cmd/demo/node.go

```diff
@@ -122,6 +125,17 @@ func importAccount(secret string) (*ewallet.Wallet, *ewallet.Account, error) {
 	return wallet, acc.(*ewallet.Account), err
 }
 
+func getOnChainBal(ctx context.Context, addressList ...wallet.Address) (balList []*big.Int, err error) {
```

Comment:
Don't use superfluous words in variable names like `fooList`, just call it `fs` or `foos`.

Reply:
Addressed in this commit - 13f6ee6.